tests/foreman/rhai/test_rhai.py

```diff
@@ -120,14 +120,17 @@ def test_unregister_system_from_rhai(self):
                 self.ak_name,
                 self.org_label,
             )
+
             with Session(self.browser) as session:
                 session.nav.go_to_select_org(self.org_name)
                 Navigator(self.browser).go_to_insights_systems()
                 # Click on the unregister icon 'X' in the table against the
                 # registered system listed.
-                session.nav.click(
-                    locators['insights.unregister_system']
-                )
+                session.nav.wait_for_ajax(timeout=40)
+                hostname = self.rhai.hostname.stdout[0]
```

Comment:
Having a shared hostname on rhai module will make difficult if we start to run tests in parallel. Also this information makes sense to be on the virtual machine itself (it is already available though at vm.hostname).